development/tools/jobs/kyma/end_to_end_backup_restore_test_test.go

```diff
@@ -0,0 +1,66 @@
+package kyma
+
+import (
+	"testing"
+
+	"github.com/kyma-project/test-infra/development/tools/jobs/tester"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEndToEndBackupRstoreTestJobsPresubmit(t *testing.T) {
+	// WHEN
+	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/tests/end-to-end/backup-restore-test/backup-restore-test.yaml")
+	// THEN
+	require.NoError(t, err)
+
+	assert.Len(t, jobConfig.Presubmits, 1)
+	kymaPresubmits, ex := jobConfig.Presubmits["kyma-project/kyma"]
+	assert.True(t, ex)
+	assert.Len(t, kymaPresubmits, 1)
+
+	actualPresubmit := kymaPresubmits[0]
+	expName := "pre-master-kyma-tests-end-to-end-backup-restore-test"
```

Comment:
there is a function for finding job by name: tester.FindPresubmitJobByName, thanks to that helper test can be shorter

Reply:
Using now function FindPresubmitJobByName